var/spack/repos/builtin/packages/py-sina/package.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
```

Comment:
```suggestion
# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Reply:
I built it yesterday on LC RZ

Reply:
Then there are likely missing dependencies or dependency constraints (e.g., version of Python and or the dependencies perhaps).